dtonator/src/main/java/com/bizo/dtonator/config/DtoProperty.java

```diff
@@ -147,4 +147,59 @@ public boolean isChainedId() {
     return isChainedId;
   }
 
+  @Override
+  public int hashCode() {
```

Comment:
Similar, do we still need these? If so, I'd generally prefer the apache commons HashCodeBuilder/EqualsBuilder as these Eclipse-/Intellij-generated methods are painful to read.